docs/en/operation/Basic-Logging.md

```diff
@@ -0,0 +1,208 @@
+---
+layout: global
+title: Basic Logging
+nickname: Basic Logging
+group: Operations
+priority: 11
+---
+
+* Table of Contents
+{:toc}
+
+## Introduction
+
+This page describes the basic logging function provided by Alluxio server processes (e.g., masters, workers and etc)
+and application processes utilizing Alluxio clients (e.g., Spark or MapReduce jobs running on Alluxio).
+Note that, there is an advanced feature that streams server-side and client-side logs to separate Alluxio logservers
+(see [remote logging]({{ '/en/operation/Remote-Logging.html' | relativize_url }}) for more details).
+
+Alluxio logging is implemented using [`log4j`](https://logging.apache.org/log4j/) and thus
+most of the configuration is done through `log4j.properties`.
+
+## Log Location
+
+### Server Logs
+
+Log files for each individual Alluxio server process (e.g., master, worker, FUSE, proxy) can be found under `${ALLUXIO_HOME}/logs/`,
+with file names like `master.log`, `worker.log`, `fuse.log` and `proxy.log`.
```

Comment:
comma after fuse.log

when listing 3 or more items, use the oxford comma after the second to last item
https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/

Reply:
thanks . good to know.